Sources/TuistGenerator/Utils/SettingsHelper.swift

```diff
@@ -0,0 +1,49 @@
+import Foundation
+import XcodeProj
+
+final class SettingsHelper {
+    func extend(buildSettings: inout [String: Any], with other: [String: Any]) {
+        other.forEach { key, value in
+            if buildSettings[key] == nil || (value as? String)?.contains("$(inherited)") == false {
+                buildSettings[key] = value
+            } else {
+                let previousValue: Any = buildSettings[key]!
```

Comment:
is it worth inverting the if statement so we don't need to force unwrap here?

Reply:
Thanks @ollieatkinson for looking into that. I just copied the methods over to the helper without much thinking 😅. Agree, we can simplify it a little bit, I will do it.

Reply:
In which case its not a high priority - feel free to discard the comment!